lib/fs/fs.ml

```diff
@@ -14,13 +18,17 @@ type tree =
 let binary_file_pattern = Str.regexp ".*\\[bat warning\\].*Binary.*content*."
 let binary_file_warning = "This file is binary and cannot be displayed"
 
+let bat_cmd =
+  "bat --style=numbers,changes --color=always --italic-text=always \
+   --paging=never --terminal-width=80 "
```

Comment:
Good idea to move this string outside nevertheless.

Let's also use OCaml syntax for string literals to slightly prettify it.

```suggestion
  {|bat --style=numbers,changes --color=always --italic-text=always --paging=never --terminal-width=80 |}
```